src/queueserver.ts

```diff
@@ -450,3 +450,24 @@ export const modifyQueue = async(op: QueueOps): Promise<IPlanModify> => {
     console.log(res);
     return res.data;
 }
+
+export const incrementPosition = async(item1_uid: string, item2_uid: string): Promise<IPlanModify> => {
```

Comment:
Why not to make the args the same as in the post request below (such as `uid` and `after_uid`)?

Reply:
Updated